lib/vanagon/component/source/http.rb

```diff
@@ -105,7 +105,7 @@ def extract(tar)
             end
           else
             # Extension does not appear to be an archive
-            return nil
+            return ':'
```

Comment:
I'm seeing a lot of the `:` operator being spread around; are we worried that this will paint us into an explicitly bourne-shell shaped corner?